assets/js/Line.tsx

```diff
@@ -97,26 +118,28 @@ function Line({
           readOnly={readOnly}
         />
       )}
-      {line === 'Silver' && (
-      <label className="mt-1 mb-4">
-        Chelsea Drawbridge Announcements
-        <div className="switch">
-          <input
-            name="chelsea_bridge"
-            type="checkbox"
-            className="switch-input"
-            checked={chelseaBridgeAnnouncements === 'auto'}
-            onChange={(e) => {
-              setChelseaBridgeAnnouncements(e.target.checked ? 'auto' : 'off');
-            }}
-          />
-          <span className="switch-label">Switch</span>
-        </div>
-      </label>
-      )}
+
       {!readOnly && (
-        <div className="viewer--toggle-all">
-          {batchModes.auto && (
+        <>
```

Comment:
It looks like there's an extra change here to move the bridge announcements switch inside the `!readOnly` block — I guess this is something [we want to fix](https://app.asana.com/0/881264583703207/1199112192455310) anyway, but it should probably be its own commit, if not its own PR (and have tests and stuff).

Reply:
ok i'll remove it then